lib/identity_job_log_subscriber.rb

```diff
@@ -196,7 +196,11 @@ def duplicate_cron_error?(ex)
   end
 
   def should_error?(job, ex)
-    job.class.warning_error_classes.none? { |warning_class| ex.is_a?(warning_class) }
+    if job.class.respond_to?(:warning_error_classes)
```

Comment:
seems ok to me, should we try to add a spec for it?

Another idea could be:

```suggestion
    if job.is_a?(ApplicationJob)
```

Reply:
I like that more! Spec added in e3a0eed1e